Assets/css/datatables.min.css

```diff
@@ -4,36 +4,12 @@
  *
  * To rebuild or modify this file with the latest versions of the included
  * software please visit:
- *   https://datatables.net/download/#bs/jszip-2.5.0/dt-1.10.16/b-1.5.1/b-colvis-1.5.1/b-html5-1.5.1/b-print-1.5.1/cr-1.4.1/fc-3.2.4/fh-3.1.3/r-2.2.1/rg-1.0.2/rr-1.2.3/sc-1.4.4
+ *   https://datatables.net/download/#dt/dt-1.10.16
```

Comment:
It looks like you switched to core datatables, dropping the extensions Scott selected like Bootstrap support. Any reason why?

Reply:
nope,